datagovharvester/validate/dcat_us.py

```diff
@@ -0,0 +1,31 @@
+from jsonschema.exceptions import ValidationError
+from jsonschema import Draft202012Validator
+
+
+def parse_errors(errors):
+    error_message = ""
+
+    for error in errors:
+        error_message += (
+            f"error: {error.message}. offending element: {error.json_path} \n"
+        )
+
+    return error_message
+
+
+def validate_json_schema(json_data, dataset_schema):
+    success = None
+    error_message = ""
+
+    validator = Draft202012Validator(dataset_schema)
+
+    try:
+        validator.validate(json_data)
+        success = True
+        error_message = "no errors"
+    except ValidationError:
+        success = False
+        errors = validator.iter_errors(json_data)
+        error_message = parse_errors(errors)
+
+    return success, error_message
```

Comment:
Can we somehow highlight that this is the only real core code that was added?  Add docstrings and a feature list in the README?